File: server/Models/Account.cs
namespace final.Models;

public class Profile : RepoItem<string>
{
  public string Name { get; set; }
  public string Picture { get; set; }
  public string Bio { get; set; }
  public string CoverImg { get; set; }
  public string GitHub { get; set; }
  public string LinkedIn { get; set; }

}

public class Account : Profile
{
  public string Email { get; set; }
  public string Number { get; set; }
}

File: client/src/models/Account.js
export class Account {
	constructor(data) {
		this.id = data.id;
		this.email = data.email;
		this.number = data.number;

		this.name = data.name;
		this.picture = data.picture;
		this.coverImg = data.coverImg;
		this.bio = data.bio;
		this.gitHub = data.gitHub;
		this.linkedIn = data.linkedIn;
	}
}
